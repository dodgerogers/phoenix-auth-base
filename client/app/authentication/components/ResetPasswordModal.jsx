import React from 'react'
import PropTypes from 'prop-types';
import AuthenticationModal from './AuthenticationModal';
import ResetPasswordFormContainer from '../containers/ResetPasswordFormContainer';
import AuthenticationLinks, { links } from './AuthenticationLinks';


const ResetPasswordModal = props => {
  return (
    <AuthenticationModal title="Reset your password" {...props}>
      <ResetPasswordFormContainer {...props} />
      <AuthenticationLinks exclude={[links.RESET_PASSWORD]} />
    </AuthenticationModal>
  );
}

export default ResetPasswordModal;
