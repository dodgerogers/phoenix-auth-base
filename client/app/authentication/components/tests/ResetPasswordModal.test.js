import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ResetPasswordModal from '../ResetPasswordModal';


describe('ResetPasswordModal', () => {
  const fullProps = {
    close: jest.fn(),
  };

  const pureComponent = (props) => shallow(
    <ResetPasswordModal {...props} />
  );

  it('matches snapshot', () => {
    const wrapper = pureComponent(fullProps);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
