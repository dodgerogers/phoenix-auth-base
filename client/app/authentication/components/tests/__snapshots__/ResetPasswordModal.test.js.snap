// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordModal matches snapshot 1`] = `
<AuthenticationModal
  close={[Function]}
  title="Reset your password"
>
  <Connect(ReduxForm)
    close={[Function]}
  />
  <AuthenticationLinks
    exclude={
      Array [
        "RESET_PASSWORD",
      ]
    }
  />
</AuthenticationModal>
`;
